test/e2e/custom/test_custom_model_grpc.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import json
+import os
+
+import pytest
+from kserve import (
+    KServeClient,
+    V1beta1InferenceService,
+    V1beta1InferenceServiceSpec,
+    V1beta1PredictorSpec,
+    constants
+)
+from kubernetes import client
+from kubernetes.client import V1Container, V1ContainerPort
+from kserve.grpc import grpc_predict_v2_pb2 as pb
+from ..common.utils import predict_grpc
+from ..common.utils import KSERVE_TEST_NAMESPACE
+
+
+def test_custom_model_grpc():
+    service_name = "custom-model"
+
+    predictor = V1beta1PredictorSpec(
+        containers=[
+            V1Container(
+                name="kserve-container",
+                image="suryasmart619/custom-model-grpc:v1", #TODO: Use Kserve repository
```

Comment:
I think we can build the `CustomModel` as e2e test image like how did with transformer
https://github.com/kserve/kserve/blob/master/test/scripts/gh-actions/build-server-runtimes.sh#L67